frontend/assets/javascripts/src/modules/form/validation/existEmail.es6

```diff
@@ -0,0 +1,32 @@
+import ajax from 'ajax';
+import debounce from 'lodash/debounce'
+
+export function init() {
+    const emailField = document.getElementById('email');
+    const isUseErrorMessage = document.getElementById('form-field__error-message-email-checker');
+    const checkerURL = '/user/check-existing-email';
+
+    const validateEmail = debounce((event) => {
+        debugger;
+        let prospectiveEmail = emailField.value;
+
+        if(!prospectiveEmail.includes('@') || !prospectiveEmail.includes('.')) {
```

Comment:
Ok, makes sense. I was just thinking that lots of email addresses have dots earlier on, e.g. `first name.lastname@theguardian.com`. If you're specifically checking for the TLD `.`, is it worth doing something like:

```
!prospectiveEmail.split('@')[1].includes('.')
```

I realise this is just a quick check, so don't bother if you think it's not worth it =).